docs/spec/api.md

```diff
@@ -124,6 +124,7 @@ specification to correspond with the versions enumerated here.
   <dd>
     <li>Allow repository name components to be one character.</li>
     <li>Clarified that single component names are allowed.</li>
+    <li>Specify the delete API for layers and manifests.</li>
```

Comment:
I think this has to go in 2.0.5.

I am going to change these revisions to be letters, so they aren't confused with any other versions.